src/picongpu/submit/taurus/picongpu.profile.example

```diff
@@ -3,10 +3,12 @@ module purge
 # General modules #############################################################
 #
 module load oscar-modules llview
-module load cmake/2.8.11 git
+module load cmake/3.3.1 git
 module load cuda/6.5.14  # gcc 4.8, intel 14.0 == 2013-sp1
 module load bullxmpi
 module load gnuplot/4.6.1
+module load openmpi/1.6
```

Comment:
oh crap, did not see the two updates.

`bullxmpi` & `openmpi/1.6` ? does not seem correct to me